src/components/GPUProviders.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { ClusterOverviewDetailsItem } from '@openshift-console/dynamic-plugin-sdk';
+import { useTranslation } from 'react-i18next';
+
+import { useAllGpuProviders } from '../resources/nodes';
+import { Dash } from './Dash';
+
+const GPUProviders: React.FC = () => {
+  const { t } = useTranslation('plugin__console-plugin-nvidia-gpu');
+  const [providers, loaded, loadError] = useAllGpuProviders();
+
+  if (loadError) {
+    return <Dash />;
+  }
```

Comment:
this early return is not needed as error state is handled by `ClusterOverviewDetailsItem`

Reply:
thanks, leftover